### PR TITLE
Fix "can not" spelling

### PR DESCRIPTION
**Pull request by mvorisek**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by mvorisek** [comment]
Please propose wanted fix to fabbot.io CS. Of course, I can generate some spaces with like str_repeat, but maybe the source of the output should be fixed.

File: src/Symfony/Component/Config/Definition/PrototypedArrayNode.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ public function addChild(NodeInterface $node)
     protected function finalizeValue($value)
     {
         if (false === $value) {
-            throw new UnsetKeyException(sprintf('Unsetting key for path "%s", value: %s.', $this->getPath(), json_encode($value)));
+            throw new UnsetKeyException(sprintf('Unsetting key for path "%s", value: "%s".', $this->getPath(), json_encode($value)));
```
Comment (chalasr) [inline src/Symfony/Component/Config/Definition/PrototypedArrayNode.php]: Not sure about this one, I would not put quotes around a boolean.
If it was reported by fabbot, please revert. Sometimes he is wrong for such cases

**Review by chalasr — APPROVED** [review]
Looks good to me.

File: src/Symfony/Component/Config/Definition/PrototypedArrayNode.php (hunk 1 above)
Comment (mvorisek) [reply src/Symfony/Component/Config/Definition/PrototypedArrayNode.php]: done

Comment (chalasr) [reply src/Symfony/Component/Config/Definition/PrototypedArrayNode.php]: thanks

**Review by phansys — APPROVED** [review]

**Comment by OskarStark** [comment]
This can be a job for fabbot

cc @fabpot

**Review by derrabus — APPROVED** [review]

**Comment by fabpot** [comment]
even if "cannot" is more common, "can not" is perfectly valid English, not sure we want to do this change.

**Comment by mvorisek** [comment]
Symfony does already use "cannot" - https://github.com/symfony/symfony/search?q=cannot , this PR unifies the other uses.

Citing from https://www.grammarly.com/blog/cannot-or-can-not/ grammar:
> - In formal writing and where contractions are frowned upon, use cannot.
> - It is possible to write can not, but you generally find it only as part of some other construction, such as “not only . . . but also.”

**Review by nicolas-grekas — APPROVED** [review]
This is typically the kind of PR that can create way more burden than what they're worth. I had the same hesitations as @fabpot, but since this is apparently more correct, I'm ok to do it and move on quickly.

**Comment by nicolas-grekas** [comment]
Thank you @mvorisek.

**Pull request merged**